### DISCUSSION TITLE
Is this tool the same as Delphi official migration tool ? (migrationtool)

### DISCUSSION THREAD

**Discussion by sfajardo2016** [alefragnani/delphi-james#24] [Q&A] [answered]
Is this tool the same as Delphi official migration tool ? (migrationtool)

Found here

[https://docwiki.embarcadero.com/RADStudio/Sydney/en/Settings_Migration_Tool](url)

**Comment by alefragnani** [comment, marked as answer]
Hi @sfajardo2016 ,,

No, it isn’t, and both serves two different purposes.

James is designed for those that like/need to have something like an isolated profile for each project you work. For every project you can have different settings and installed components, with an easy to use switch approach.

The Migration Tool, on the other hand, is used to migrate your setting from one Delphi version to another, when you install multiple versions.

Hope this helps
